implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -273,6 +274,14 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "smallrye.jwt.verify.aud")
     Optional<Set<String>> expectedAudience;
 
+    /**
+     * List of claim names that must be present in the JWT for it to be valid. The configuration should be specified
+     * as a comma-separated list per MP Config requirements for a collection property.
```

Comment:
@radcortez Lets drop ` per MP Config requirements for a collection property` - we already have few properties like this one where we don't justify why (earlier Set would not even work). And please update README with the same description